alpha/external-web/app/routes/application-submitted.js

```diff
@@ -1,6 +1,41 @@
 var router = require('../routes')
+var client = require('../services/eligibility-client')
+var logger = require('../services/bunyan-logger')
 
-router.get('/application-submitted', function (request, response, next) {
-  response.render('application-submitted')
+const tasksCollection = 'tasks'
+const claimantsCollection = 'claimants'
+
+router.get('/application-submitted/:claimant_id', function (request, response, next) {
```

Comment:
creating notification should be done at last POST before application is completed.
A GET request should be Idempotent and repeatable.